#This needs a slight patch for 2.6 currently because of a missing API. This is better than doing it with reflection.
Index: basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java
===================================================================
--- basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java (revision 73061)
+++ basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java (working copy)
@@ -503,7 +503,7 @@
                    Site site = SiteService.getSite(toolConfig.getSiteId());
                    SitePage page = site.getPage(toolConfig.getPageId());
            page.setTitle(imsTIPageTitle);
-           page.setTitleCustom(true);
+//         page.setTitleCustom(true);
            SiteService.save(site);
            } catch (Exception e) {
                        setErrorMessage(request, rb.getString("error.page.title"));
Index: pom.xml
===================================================================
--- pom.xml (revision 73061)
+++ pom.xml (working copy)
@@ -2,11 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     
+    <!--
     <parent>
         <groupId>org.sakaiproject.purepoms</groupId>
         <artifactId>sakai-standard-tool</artifactId>
         <version>2.7.0-build04</version>
     </parent>
+    -->
     
     <name>BasicLTI base pom (basiclti)</name>
     <groupId>org.sakaiproject.basiclti</groupId>
@@ -22,13 +24,11 @@
     <!-- Change this to the Sakai released version you're running -->
     <!-- Unreleased or snapshot tags may require the entire source -->
     <!-- example: <version>2.6.1</version> -->
-    <!--
     <parent>
         <artifactId>master</artifactId>
         <groupId>org.sakaiproject</groupId>
-        <version>2.7.0-SNAPSHOT</version>
+        <version>2.6-SNAPSHOT</version>
     </parent>
-    -->
 
     <properties>
         <url.localsite>scp://source.sakaiproject.org/var/www/html/release/basiclti/${project.version}</url.localsite>
